grader/self.py

```diff
@@ -263,7 +263,10 @@ def check_lock() -> List[Check]:
                         success_criteria='Hello World!    ' * 8) + \
         check_execution('./selfie -c selfie.c -m 128 -c <assignment>print-with-lock.c -y 10',
                         '16 processes are printing in sequential order with the use of locks on HYPSTER',
-                        success_criteria='Hello World!    ' * 8)
+                        success_criteria='Hello World!    ' * 8) + \
+        check_execution('./selfie -c <assignment>release-after-exit.c -m 128',
```

Comment:
Is it easily possible to set a separate timeout for this test? (So that it doesn't take too long in case a student didn't implement the implicit unlock upon exit.)

Reply:
I now explicitly set the timeout to 5 seconds. This should suffice as the check's source is rather small.